src/Any/Signer.cpp

```diff
@@ -74,6 +77,57 @@ Any::Proto::SigningOutput Any::Signer::sign() const noexcept {
             }
             break;
         }
+        case TWCoinTypeIoTeX: {
```

Comment:
if we want to support Tomochcain, should we add case `case TWCoinTomoChain` right after `case TWCoinEthereum` ?

Reply:
maybe we should consider to use `BlockchainType` ? instead of `CoinType`